src/SprykerEco/Zed/Payone/Business/RiskManager/RiskCheckManager.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\RiskManager;
+
+use Generated\Shared\Transfer\AddressCheckResponseTransfer;
+use Generated\Shared\Transfer\ConsumerScoreResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Payone\Business\RiskManager\Factory\RiskCheckFactoryInterface;
+use SprykerEco\Zed\Payone\Business\RiskManager\Mapper\RiskCheckMapperInterface;
+
+class RiskCheckManager implements RiskCheckManagerInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\RiskManager\Mapper\RiskCheckMapperInterface
+     */
+    protected $riskCheckMapper;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface
+     */
+    protected $executionAdapter;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\RiskManager\Factory\RiskCheckFactoryInterface
+     */
+    protected $riskCheckFactory;
+
+    /**
+     * @param \SprykerEco\Zed\Payone\Business\RiskManager\Mapper\RiskCheckMapperInterface $riskCheckMapper
+     * @param \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface $executionAdapter
+     * @param \SprykerEco\Zed\Payone\Business\RiskManager\Factory\RiskCheckFactoryInterface $riskCheckFactory
+     */
+    public function __construct(RiskCheckMapperInterface $riskCheckMapper, AdapterInterface $executionAdapter, RiskCheckFactoryInterface $riskCheckFactory)
+    {
+        $this->riskCheckMapper = $riskCheckMapper;
+        $this->executionAdapter = $executionAdapter;
+        $this->riskCheckFactory = $riskCheckFactory;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\AddressCheckResponseTransfer
+     */
+    public function addressCheckRequest(QuoteTransfer $quoteTransfer): AddressCheckResponseTransfer
+    {
+        $requestContainer = $this->riskCheckMapper->mapAddressCheck($quoteTransfer);
+        $response = $this->executionAdapter->sendRequest($requestContainer);
+
+        $responseContainer = $this->riskCheckFactory->createAddressCheckResponseContainer();
+        $responseContainer->init($response);
+
+        $addressCheckResponseTransfer = new AddressCheckResponseTransfer();
+        $addressCheckResponseTransfer->setStatus($responseContainer->getStatus());
+        if ($responseContainer->getCustomermessage()) {
```

Comment:
please use `isset`.

Reply:
isset() only works with variables as passing anything else will result in a parse error. For checking if constants are set use the defined() function.

is it a real reason to use isset here?